judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/problem/ContestProblemResource.java

```diff
@@ -218,6 +222,8 @@ public ContestProblemWorksheet getBundleProblemWorksheet(
         String problemJid = problem.getProblemJid();
         ProblemInfo problemInfo = clientProblemService.getProblem(sandalphonClientAuthHeader, problemJid);
 
+        boolean canManage = problemRoleChecker.canManage(actorJid, contest);
```

Comment:
Actually, I think this check is not required at all. We can just always return the items without grading info from uriel. Even managers need not that info. If that info is required, it should be fetched from the source of truth (sandalphon), not via Uriel.

Reply:
Changed